### PR TITLE
Todo list - Unit testing part 1

### PR DESCRIPTION
**Pull request by Kossi-stack**
- We created a new branch 
- We set up babel configuration
- We tested add new task
- We tested remove task
- We fixed linters errors
- We added descriptive readme content

### CODE REVIEW

File: tests/addAndRemove.test.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  test('Delete task with index 2', () => {
+    removeTodo(todos, 2);
+    expect((todos)).toHaveLength(3);
+  });
+
+  test('Update index after task deletion', () => {
+    expect(todos[2].index).toBe(3);
+  });
+});
```
Comment (Emmyn5600) [inline tests/addAndRemove.test.js:60]: - [ ] Great Implementation, all tests are passing successfully, however, let us add few tests to for both Add and remove function
arr5

`For Adding function`

Adds a task to a nonempty list; check description and index
Adds a task to an empty list; check description

`For Deleting func`

Update localStorage after deleting  elements
Update dom after deleting compeleted elements

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @Kossi-stack  and @codecaiine 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- All tests are passing successfully:heavy_check_mark:

Changes required :writing_hand: 


      * #### Please check the comments under the review.
       
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: tests/addAndRemove.test.js (hunk 1 above)
Comment (codecaiine) [reply tests/addAndRemove.test.js:60]: Hi Emmyn ,
Thank you for your comment but.
But there are some parts we don't understand cause I think we are implemented all the requirements and everything works fine.
 
demo2
demo-1

Can you please explain to us what were are supposed to do again, cause we don't understand your comments?
Cheers,

Comment (codecaiine) [reply tests/addAndRemove.test.js]: We updated the project with jet.mock . You can check with the screenshot below. 

moock

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @Kossi-stack @codecaiine 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**